app/javascript/application.js

```diff
@@ -1,3 +1,5 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails"
 import "controllers"
+import "popper"
+import "bootstrap"
```

Comment:
add EOL.

Reply:
you have marked this as resolved but I can't see any EOL here.

Reply:
sorry bhaiya, somehow the change got  reverted mistakenly. I have done it again bhaiya